src/actions/actions.js

```diff
@@ -0,0 +1,56 @@
+const databaseName = 'todoz'
+const databaseName_test = 'todoz_test'
+let connectionString
+
+if(process.env.NODE_ENV === 'development'){
+  connectionString = `postgres://${process.env.USER}@localhost:5432/${databaseName}`
+} else if (process.env.NODE_ENV === 'test'){
+  connectionString = `postgres://${process.env.USER}@localhost:5432/${databaseName_test}`
+}
+
+// const pgp = require('pg-promise')();
+// const db = pgp( connectionString );
+const db = require('../database/db.js')
+
+
+const getAllTodos = () => {
+  return db.any(`SELECT id, description FROM todos
+    WHERE is_completed = false`)
+}
+
+
+const getCompletedTodos = () => {
+  return db.any(`SELECT * FROM
+    todos WHERE is_completed = true`)
+
+}
```

Comment:
Since you structured `getAllTodos` to only select id, and description make this one uniform with that one for consistency. You can either make this one only SELECT those two fields or make your `getAllTodos` a `SELECT *`